### ISSUE TITLE
Python lessons don't mention the ':' character and its importance

### ISSUE THREAD

**Issue by gkiar** [swcarpentry/python-novice-inflammation#514]
I've noticed in both the loops and conditionals lessons that no explanation of the syntax for these operators is explicit about including the `:` character at the end of the declaration - this is an important distinction from other languages, and perhaps not obvious.

**Comment by maxim-belkin** [comment]
I agree, mandatory nature of colon warrants at least a brief word about it. Would you like to submit a PR?

**Comment by gkiar** [comment]
Yep! See #516
